pkg/errors/errors.go

```diff
@@ -18,6 +18,7 @@ const (
 	KindRateLimit      = http.StatusTooManyRequests
 	KindNotImplemented = http.StatusNotImplemented
 	KindRedirect       = http.StatusMovedPermanently
+	KindConfigError    = -1
```

Comment:
I believe the HTTP protocol gives you some room for custom codes
In this case this is a client (user) error so maybe we can give it a code between 451 and 499: https://golang.org/pkg/net/http/#pkg-constants

It doesn't really matter but just being a bit more conventional :)

Reply:
422 / `http.StatusUnprocessableEntity` is a common response when a client provide syntactically valid but _semantically invalid_ objects to an API and could be appropriate here, if you want to stick with valid HTTP codes.

Reply:
@marwan-at-work fair enough! I'll take @bluekeyes's suggestion and go with 422, just so we can use a constant.